packages/commerce-sdk-react/README.md

```diff
@@ -54,6 +54,10 @@ import {CommerceApiProvider} from '@salesforce/commerce-sdk-react'
 import {withReactQuery} from '@salesforce/pwa-kit-react-sdk/ssr/universal/components/with-react-query'
 
 const AppConfig = ({children}) => {
+    const headers = {
+        'correlation-id': correlationId
```

Comment:
May be add comment explaining the need  for correlation id

Reply:
This bit was just me copying the code that we already have inside app-config in the template-retail-react-app and the generator so that the example more closely aligns with the actual code that developers will see.